rotkehlchen/data_migrations/manager.py

```diff
@@ -32,7 +32,9 @@ def maybe_migrate_data(self) -> None:
         current_migration = settings.last_data_migration
         for migration in MIGRATION_LIST:
             if current_migration < migration.version:
-                self._perform_migration(migration)
+                if self._perform_migration(migration) is False:
```

Comment:
@yabirgb If a migration fails we STOP. Before we were having failed migrations and the code was happily continuing onto the next one and also writing to the DB that migration was complete.